content/post/2023-08-12-rebrand.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Our Rebranding
+date: 2023-08-12
+author: NUS Hackers
+url: /2023/08/12/rebrand
+
+---
+
+We have decided to bid farewell to Barcode Bob and welcome Quackers!
+
+<div style="display: flex; justify-content: space-between; margin-top: 20px; margin-inline: 120px; ">
+  <img src="/img/2023/barcode_bob_head.png" alt="Barcode Bob head" style="width: 240px;">
+  <img src="/img/2023/logo_head_light.svg" alt="Quackers looking at you" style="width: 105px;" />
+</div>
+
+As our gallery of events has grown over the years, we increasingly felt the need for unified branding, from Friday Hacks to Hack&Roll.
+
+Expanding our community made it essential to define our shared focus clearly - which is bringing people together and inspiring each other to build something.
+
+We extend this invitation to join our community to anyone interested in hacking and building, whether you're a seasoned hacker who has created a game in assembly or someone starting their first website. We would love to have you at our events!
+And for those who may not have read this post, our new branding aims to convey just that.
+
+<div style="display: flex; justify-content: center; padding: 20px; padding-top: 5px;">
+  <img src="/img/2023/logo_light.svg" alt="Our new logo featuring Quackers with the wordmark" style="width: 400px;" />
+</div>
+
+<img src="/img/2023/lego_guy.png" alt="Lego guy waving" style="float: right; width: 180px; height: auto;" />
+Our new logo features a headshot of our mascot, Quackers, whose head is inspired by one of our favorite childhood toys - Lego! Much like Lego, we hope our events will inspire people to exercise their creativity and have fun building things that may even be outside of the instruction manual.
+
+<div style="height: 50px;"></div>
+
+We also hope Quackers is someone you can trust to listen to the most unspeakable bugs in your code, just like the rubber duck sitting on some of your desks. In fact, Quackers has seen so much that its eyes oddly resemble text on the terminal.
+
+Lastly, we wanted to stick with what has become familiar to our hackers' brand - black with a splash of orange. Retaining this color scheme, which has made Barcode Bob recognizable, was important to us.
+
+We also took this chance to unify the branding of our core events, Friday Hacks, Hackerschool, and Hackers Toolbox (Formerly Hacker Tools). We hope this will make it easier for you to identify a Hackers™ event.
```

Comment:
```suggestion
We also took this chance to unify the branding of our core events, Friday Hacks, Hackerschool, and Hackers Toolbox (formerly Hacker Tools). We hope this will make it easier for you to identify a Hackers event.
```

No strong opinions on the following but let me know what you think!

1. nit but I think "formerly" shouldn't be capitalized? 
2. I feel like you included the trademark as a joke but we aren't trademarked so we probably shouldn't put it in?

Reply:
sounds good!